File: app/src/main/java/com/github/android/lvrn/lvrnproject/view/dialog/notebookselection/NotebookSelectionDialogFragment.java
package com.github.android.lvrn.lvrnproject.view.dialog.notebookselection;

import com.github.valhallalabs.laverna.persistent.entity.Notebook;

/**
 * @author Vadim Boitsov <vadimboitsov1@gmail.com>
 */

public interface NotebookSelectionDialogFragment {

    void updateRecyclerView();

    void setSelectedNotebook(Notebook notebook);

}

File: app/src/main/java/com/github/android/lvrn/lvrnproject/view/dialog/notebookselection/NotebookSelectionDialogFragment.kt
package com.github.android.lvrn.lvrnproject.view.dialog.notebookselection

import com.github.valhallalabs.laverna.persistent.entity.Notebook

/**
 * @author Vadim Boitsov <vadimboitsov1></vadimboitsov1>@gmail.com>
 */
interface NotebookSelectionDialogFragment {
    fun updateRecyclerView()

    fun setSelectedNotebook(notebook: Notebook?)
}
